tests/blocks/test_core.py

```diff
@@ -1573,6 +1573,18 @@ class A(Block):
 
         assert A.get_description() == "But I will"
 
+    def test_no_griffe_logs(self, caplog):
```

Comment:
Could you use the `capsys` fixture in this test also to be _**extra**_ sure that we are printing anything we shouldn't be?